dashboard/lib/controllers/batchQueue/index.js

```diff
@@ -26,11 +26,11 @@ class BatchQueue {
             await prevPromise;
             await this.queueProcessor(item);
 
-            // at last item, mark processing as complete so that the new batch can be saved
-            if (index == currentQueue.length - 1) { 
-                this.processing = false
+            // at last, mark processing as complete so that new batch is saved
+            if (index == currentQueue.length - 1) {
```

Comment:
Nitpick: Use `===`. Should enforce in eslint.

Reply:
Oops, will fix this asap.